src/containers/filter/index.js

```diff
@@ -113,6 +113,8 @@ export default class FilterProvider extends PureComponent {
           removeFilter={this.removeFilter}
           selectedFilters={this.state.selectedFilters}
           updateSelectedFilters={this.updateSelectedFilters}
+          downloadExcel={this.props.downloadExcel}
+          addRecord={this.props.addRecord}
```

Comment:
I think we should not hardcode these here instead allow users to render custom content?